siege_utilities/geo/gazetteers/census_gazetteer.py

```
@@ -0,0 +1,321 @@
+        # _geoid) keeping the first occurrence.
+        seen: set = set()
+        deduped: list = []
+        for row in rows:
+            key = (row["_layer"], row["_geoid"])
+            if key in seen:
+                continue
+            seen.add(key)
+            deduped.append(row)
+        return tuple(deduped)
+
+    def _fetch_polygon(self, layer: int, geoid: str) -> Any:
```

Ruff (0.15.12)

[warning] Dynamically typed expressions (typing.Any) are disallowed in `_fetch_polygon`

(ANN401)